.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "relic"]
 	path = relic
-	url = https://github.com/jhunkeler/relic.git
+	url = https://github.com/spacetelescope/relic.git
```

Comment:
Hah!

Reply:
Yeah it happened after several years :grimacing: